content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,462 @@
+
+A new mode, RGB parade, appears and displays waveforms that represent the levels of each layer (red, green and blue). This mode allows you to visualize how the color components of a part of the image are distributed.
+
+This mode could be displayed by select histogram display in wave mode (with the top left icon on the histogram) and then by changing the wave mode with the slighlty next one just on the right, to go on RGB parade mode.
+
+  <div style="text-align: center">
+![RGB parade mode]({filename}en/rgb-parade.png)
+  </div>
+
+Add to that an important code rewriting of the histogram for better performances.
+
+## Drawn mask : curved gradient
+
+When adding a drawn mask, it is now possible to bend the gradient shape (gradient). To do this, point the mouse cursor on the gradient bar and scroll up or down with the wheel (or touchpad), depending on the desired curvature. `Ctrl` + `scrolling` allows you to set the opacity and `Shift` + `scrolling` manages the hardness.
+
+## An improved history
+
+The history behaviour has been improved according to different users feedback. Thus, when compressing the history or loading a new image, the history will now behave this way :
+
+- any active module that cannot be deactivated will always appear first in the history stack;
+- any module that is enabled by default but can be disabled, then appears in the history stack;
+- finally, all modules activated by presets or as a result of a preference setting (workflow or sharpness selected in the preferences) are added.
+
+## And more :
+
+- Add support of AVIF file format (>= 0.7). This require to have [libavif](https://github.com/AOMediaCodec/libavif) library installed.
+- Add optional grey-scale export of TIFF for monochrome images.
+- It is now possible to export masks in TIFF format.
+- It is now possible to use more than 500 images in tethered control, which is needed for time-lapse.
+- Applying a style now supports _append_ mode (added to the existing image history) or in _overwrite_ mode (so replace the existing image history). This makes the style module consistent with _development_ module.
+- It is possible to handle (deleting, applying or exporting) multiple styles in the style module.
+- Implement undo/redo for orientation changes from the lighttable.
+- By using `Ctrl` + `click` on mask shapes, it is possible to allow the continuous creation of several similar shapes.
+- Rejecting an image keeps in memory the previously defined number of stars. If the rejection is canceled, the image will be returned to the defined number of stars.
+
+This list is not exhaustive. Some other minor improvements could be discovered by reading the complete darktable release notes: https://github.com/darktable-org/darktable/releases/tag/release-3.2.0
+
+# A future 3.4 on its way
```

Comment:
Maybe just change to "3.4 is on its way"

Reply:
Thanks but I will keep that one.

Reply:
How about "A future version 3.4 is on its way"?

Reply:
It's good but I really like the way this title is written.

Reply:
Ok I don't think it reads well in English (there's no verb and 3.4 is not really a noun) but I know when to quit ;-)